### PR TITLE
Feature - limit drag between minimum and maximum value

### PR DESCRIPTION
**Pull request by robsonsr**
I needed a feature that didn't exist, limiting the slip between the minimum and maximum value, example:
0, 1, 2, 3, 4, 5, 6, 7;
the limit passed as a parameter = 4;
the user can only drag from 0, 1, 2, 3, 4, but 5, 6, 7 are still present.

### CODE REVIEW

**Comment by ptomasroos** [comment]
Thank you for the contribution! I will take a look at it :)

**Comment by ptomasroos** [comment]
May I ask in which use-case you're using the feature? Just to understand it :)

**Comment by robsonsr** [comment]
example:
image
Today is Monday, I can only drag until Monday.
If it was Tuesday, I could only drag until Tuesday.
If it was Wednesday, I could only drag until Wednesday.

I search for data from the past and the present day, but never from the next days;

**Comment by ptomasroos** [comment]
Ah I see! Cool. Thanks

File: MultiSlider.js [hunk 1]
```diff
@@ -140,6 +141,11 @@ export default class MultiSlider extends React.Component {
   };
 
   moveOne = gestureState => {
+    if (this.props.moveLimitOne === this.props.values[0]) {
+      if (gestureState.dx > 0)
```
Comment (ptomasroos) [inline MultiSlider.js:145]: This needs to work with vertical too. Check in the file for a example how to check for dy.

Comment (Tushar321321) [reply MultiSlider.js:145]: it's not working my side please explain more about it how to use and all

**Pull request closed without merging**